src/lib.rs

```diff
@@ -18,10 +18,21 @@
 ///     finish_data(&a.bytes);
 /// }
 /// ```
-extern crate wee_alloc;
 
-#[global_allocator]
-static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+#[macro_use]
+extern crate cfg_if;
+
+cfg_if! {
```

Comment:
I am pretty sure we don't need the `cfg_if` dependency here. We can do something like `#[cfg(feature = "wee_alloc")]  static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;`

Reply:
It also has/needs that `#[global_allocator]` annotation. Will that work?

Reply:
I think so, attributes apply to the item below. If the item is not compiled in, then the attribute isn't either.